README.md

```diff
@@ -1,7 +1,10 @@
 # Schematic
 [![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FSage-Bionetworks%2Fschematic%2Fbadge%3Fref%3Ddevelop&style=flat)](https://actions-badge.atrox.dev/Sage-Bionetworks/schematic/goto?ref=develop) [![Documentation Status](https://readthedocs.org/projects/sage-schematic/badge/?version=develop)](https://sage-schematic.readthedocs.io/en/develop/?badge=develop) [![PyPI version](https://badge.fury.io/py/schematicpy.svg)](https://badge.fury.io/py/schematicpy)
 
-# Table of contents
+# TLDR
+Under Construction.
+
+# Table of Contents
```

Comment:
Small tip. I use this vscode extension to automatically create/update the table of contents on save of the file:
https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

You might find it useful as you're likely adding more sections.

Reply:
nifty. thanks!